src/components/pages/PostPage.tsx

```diff
@@ -425,8 +426,18 @@ class PostPage extends React.Component<Props, State> {
       });
     }
 
-    // Find any buttons and replace them with Material UI button
-    if (classNames.indexOf("wp-block-button") > -1) {
+    // Find any add new event buttons and make them trigger add event dialog
+    // TODO: move add event dialog into separate component from WelcomePage.tsx that can be launched from anywhere
+    if (classNames.indexOf("new-event-button") > -1) {
+      const childNode = node.children && node.children.length ? node.children[0] : null;
+      if (childNode) {
+        return (
+          <Button onClick={ this.addEvent } className={ classes.button } color="primary" variant="outlined" startIcon={ <Add /> }>
```

Comment:
<i class='graphite__hidden'>[Re: lines 435 to 435]<br/></i>

```suggestion
          <Button
            onClick={ this.addEvent }
            className={ classes.button }
            color="primary"
            variant="outlined"
            startIcon={ <Add /> }
          >
```<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/Metatavu/mantyharju-wp-theme/321?utm_source=unchanged-line-comment">Graphite</a>.</span>